template-processors/openshift-provision/files/processParameters.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Process parameters
+  hosts: localhost
+  gather_facts: false
+  tasks:
+  - set_fact:
+      _cluster_definition_dir: "{{ openshift_cluster_config }}/cluster/{{ cluster_name }}"
+
+  # Build the hierarchy for your configuration defined for a given cluster
+  - include_vars:
+      file: '{{ _cluster_definition_dir }}/vars/main.yml'
+
+  - name: Initialize hierarchy_paths
+    set_fact:
+      hierarchy_paths: ["{{ openshift_cluster_config }}/default"]
+
+  - name: Define dynamic hierarchy_paths
+    set_fact:
+      hierarchy_paths: "{{ hierarchy_paths }} + [ '{{ openshift_cluster_config }}/{{ item.key }}/{{ item.value }}' ]"
+    with_dict: "{{ hierarchy }}"
```

Comment:
It would be great to have tests verifying that the overriding here works as intended

Reply:
I will be adding the tests for this and an example for this very soon. I wanted to get the initial code out there so people could start looking in advanced.